src/shims/dir.props

```diff
@@ -15,14 +15,18 @@
     <NetFxReference Include="System" />
     <NetFxReference Include="System.Core" />
     <NetFxReference Include="System.ComponentModel.Composition" />
+    <NetFxReference Include="System.ComponentModel.DataAnnotations" />
     <NetFxReference Include="System.Data" />
     <NetFxReference Include="System.Drawing" />
     <NetFxReference Include="System.IO.Compression.FileSystem" />
+    <NetFxReference Include="System.Net" />
```

Comment:
We should look at this as I don't think it has any APIs. However we might have the type-forwards but we should double check.